core/templates/dev/head/components/top_navigation_bar/top_navigation_bar_directive.html

```diff
@@ -92,7 +92,7 @@
              from appearing off-center because 'A' is slightly thinner than 'M' in
              this font.
             -->
-            <span ng-if="isAdmin" class="oppia-navbar-role-text" style="right: 4px;">A</span>
+            <span ng-if="isAdmin" class="oppia-navbar-role-text" style="right: 4px;font-size: 11px;">A</span>
```

Comment:
Add space before font-size. 

This is the only code nit; deferring to @kevinlee12 to verify UI.

Reply:
Done!